helix/transforms/tigress/__init__.py

```diff
@@ -0,0 +1,13 @@
+import logging
```

Comment:
This would hijack any existing logging configuration - I think we should remove this logger setup in favor of https://github.com/helix-datasets/helix/issues/26